org.civicrm.bootstrapcivicrm/gulpfile.js

```diff
@@ -1,20 +1,34 @@
 var gulp = require('gulp');
 var bulk = require('gulp-sass-bulk-import');
-var rename = require('gulp-rename');
 var sass = require('gulp-sass');
+var postcss = require('gulp-postcss');
+// The "postcss-prefix-selector" uses ES6 promises
+require('es6-promise').polyfill();
+// Used to add necessary prefixes at the css level
+var postcssPrefix = require('postcss-prefix-selector');
+// Used to remove duplicated css rules
+var postcssDiscardDuplicates = require('postcss-discard-duplicates');
 
-gulp.task('sass', function () {
-  gulp.src('scss/*.scss')
+gulp.task('sass-bootstrap', function () {
+  gulp.src('scss/bootstrap/bootstrap.scss')
     .pipe(bulk())
     .pipe(sass({ outputStyle: 'compressed' }).on('error', sass.logError))
-    .pipe(rename(function (path) {
-        path.basename = 'bootstrap-civicrm-' + path.basename;
-    }))
+    .pipe(gulp.dest('css/'));
+});
+
+gulp.task('sass-civicrm', function () {
+  gulp.src('scss/civicrm/civicrm.scss')
+    .pipe(bulk())
+    .pipe(sass({ outputStyle: 'compressed' }).on('error', sass.logError))
+    .pipe(postcss([postcssPrefix({
+      prefix: '.crm-container ',
+      exclude: [/logged-in/, /crm-container/]
+    }), postcssDiscardDuplicates]))
     .pipe(gulp.dest('css/'));
 });
 
 gulp.task('watch', function () {
-    gulp.watch('scss/**/*.scss', ['sass']);
+  gulp.watch('scss/**/*.scss', ['sass', 'sass-civicrm']);
```

Comment:
You're still referencing `'sass`' here. Come to think of it, maybe you could make a `'sass'` task that simply calls both `'sass-bootstrap'` and `'sass-civicrm'`